account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/core/TekManager.java

```diff
@@ -153,4 +157,7 @@ private HpkePrivateKey toHpkePrivateKey(HpkeParameters hpkeParams, KeysetHandle
 
     }
 
+    private TekManager() {
```

Comment:
In the code above this: avoid the use of the keyword `assert`. JVM settings can choose to ignore assert statements. The original intention was that you could add a bunch of checks in dev mode and remove them for production to make it faster, but in practice it means that you get code which behaves differently in different environments. I'd recommend never using the `assert` keyword, and instead use standard `if (condition) { throw new SomeError() }` blocks instead.

Reply:
done